### PR TITLE
Omit run.py from test coverage for source Sendgrid, Sentry, Zendesk Chat

### LINKED ISSUE #35733: Sources failing 90% test threshold due to including run.py in coverage report
### Topic

_No response_

### Relevant information

Multiple sources are failing on < 90% unit test coverage due to the inclusion of the boilerplate setup code in run.py.

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What

Resolves #35733. Sources Sendgrid, Sentry and Zendesk Chat are all failing the 90% test coverage threshold due to the inclusion of the boilerplate code in run.py. Omitting run.py from the coverage report addresses the issue

### CODE REVIEW

File: airbyte-integrations/connectors/source-sendgrid/metadata.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: fbb5fbe2-16ad-4cf4-af7d-ff9d9c316c87
-  dockerImageTag: 0.4.3
+  dockerImageTag: 0.4.4
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-sendgrid/metadata.yaml:13]: I think we can just skip pumping version here and just /approve-and-merge with comment after all other CI tests passed

Comment (ChristoGrab) [reply airbyte-integrations/connectors/source-sendgrid/metadata.yaml]: Gotcha thanks! Reverted all the version bumps

**Review by octavia-approvington — APPROVED** [review]

**Comment by octavia-approvington** [comment]
Lets do it! <br/> giddddy-up

**Pull request merged**